infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/ElementListing.kt

```diff
@@ -92,7 +108,27 @@ fun toElementListing(
         .map { element -> toElementListing(context, getTransformation, plan, alignment, element) }
 }
 
-fun toElementListing(
+private fun toElementListing(
+    context: GeocodingContext?,
+    trackNumberId: IntId<TrackLayoutTrackNumber>,
+    segment: LayoutSegment,
+) = ElementListing(
```

Comment:
Ei mitään mergailua estävää, mutta kun kyseesä on selkeästi `MISSING_SECTION`:in luonti, niin sen voisi eriyttää nimeämisen osalta muista `toElementListing`:eistä (joku `toMissingElementListing` tms.)